webui/react/src/pages/InteractiveTask.tsx

```diff
@@ -83,9 +83,8 @@ export const InteractiveTask: React.FC = () => {
           }
         }
       } catch (e) {
-        handleError({
-          error: e,
-          message: 'failed querying for command state',
```

Comment:
this is a case where we were doing handleError with an object literal and it was being ignored... resulting in "unknown error"

here we're solving it in two ways at once by doing this here and changing error.ts too... but oh well.